.nucleus.yaml

```diff
@@ -3,10 +3,77 @@ core-deploy:
   project-modules:
     lwc: lwc.version
 branches:
+  release:
+    pull-request:
+      workflow: release
+      auto-start: true
+      auto-start-from-forks: false
+      # release branch should always be exactly in sync with master branch (linear history)
+      merge-method: force-push
+      required-downstream-deps:
+        # sadly these need to be repeated for every branch
+        - automation-platform/ui-interaction-explorer-components
+        - communities/microsite-template-marketing
+        - communities/shared-experience-components
+        - communities/ui-b2b-components
+        - communities/ui-cms-components
+        - communities/ui-feeds-components
+        - communities/ui-lightning-community
+        - lwc/lwc-platform
+        - salesforce/lds-lightning-platform
+        - salesforce/luvio
+        - salesforce/lwr
+        - salesforce/lwr-recipes
+        - salesforce/utam-docs
+        - salesforcedevs/developer-website
+        - uiplatform/nucleus
   ~DEFAULT~:
     pull-request:
       auto-start: true
       auto-start-from-forks: false
+      merge-method: disabled # do not auto-merge; we'll do it ourselves
+      required-downstream-deps:
+        - automation-platform/ui-interaction-explorer-components
+        - communities/microsite-template-marketing
+        - communities/shared-experience-components
+        - communities/ui-b2b-components
+        - communities/ui-cms-components
+        - communities/ui-feeds-components
+        - communities/ui-lightning-community
+        - lwc/lwc-platform
+        - salesforce/lds-lightning-platform
+        - salesforce/luvio
+        - salesforce/lwr
+        - salesforce/lwr-recipes
+        - salesforce/utam-docs
+        - salesforcedevs/developer-website
+        - uiplatform/nucleus
+  winter22:
```

Comment:
Does it mean that we have to add a new entry for each release version? I am surprised that Nucleus doesn't come with a built-in mechanism for Salesforce code lines.

Reply:
I based my code off of other `.nucleus.yaml` files. I'll ask Nucleus folks.

Reply:
Confirmed that yes, there is no wildcard option here.